libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -581,8 +588,6 @@ export class Simulator {
    */
   private stopServer() {
     this._server!.close();
-    //@ts-ignore
-    this._server!.closeAllConnections();
```

Comment:
Removing this is the only change in this PR that might address https://github.com/winglang/wing/issues/5734

If this does solve the problem, we should do further investigation as to why. Force closing connections should theoretically be safe in our tests because each test gets their own server and connections.